notebooks/Media-Pretrained/02_Training_Layer_Recap.ipynb

```diff
@@ -111,9 +87,1051 @@
+    "    print(json.dumps(create_dataset_group_response, indent=2))\n",
+    "    print ('\\nCreating the Dataset Group with dataset_group_arn = {}'.format(workshop_dataset_group_arn))\n",
+    "\n",
+    "except personalize.exceptions.ResourceAlreadyExistsException as e:  \n",
+    "    workshop_dataset_group_arn = 'arn:aws:personalize:'+region+':'+account_id+':dataset-group/'+workshop_dataset_group_name \n",
+    "    print ('\\nThe the Dataset Group eith dataset_group_arn = {} already exists'.format(workshop_dataset_group_arn))\n",
+    "    print ('\\nWe will be using the existing Dataset Group dataset_group_arn = {}'.format(workshop_dataset_group_arn))\n",
+    "    \n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Wait for Dataset Group to Have ACTIVE Status \n",
+    "\n",
+    "Before we can use the Dataset Group in any items below it must be active. This can take a minute or two. Execute the cell below and wait for it to show the ACTIVE status. It checks the status of the dataset group every 60 seconds, up to a maximum of 3 hours."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "max_time = time.time() + 3*60*60 # 3 hours\n",
+    "while time.time() < max_time:\n",
+    "    describe_dataset_group_response = personalize.describe_dataset_group(\n",
+    "        datasetGroupArn = workshop_dataset_group_arn\n",
+    "    )\n",
+    "    status = describe_dataset_group_response[\"datasetGroup\"][\"status\"]\n",
+    "    print(\"DatasetGroup: {}\".format(status))\n",
+    "    \n",
+    "    if status == \"ACTIVE\" or status == \"CREATE FAILED\":\n",
+    "        break\n",
+    "        \n",
+    "    time.sleep(60)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now that you have a dataset group, you can create a dataset for the interaction data."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Create the Interactions Schema <a class=\"anchor\" id=\"interact_schema\"></a>\n",
+    "[Back to top](#top)\n",
+    "\n",
+    "Now that we've loaded and prepared our three datasets we'll configure the Amazon Personalize service to understand our data so that it can be used to train models for generating recommendations.Amazon Personalize requires a schema for each dataset so it can map the columns in our CSVs to fields for model training. Each schema is declared in JSON using the [Apache Avro](https://avro.apache.org/) format. \n",
+    "\n",
+    "First, define a schema to tell Amazon Personalize what type of dataset you are uploading. There are several reserved and mandatory keywords required in the schema, based on the type of dataset. More detailed information can be found in the [documentation](https://docs.aws.amazon.com/personalize/latest/dg/how-it-works-dataset-schema.html).\n",
```

Comment:
Should this be: 

"There are several mandatory fields that are required in the schema, depending on the type of dataset"